configapp/configapp.py

```diff
@@ -21,16 +23,60 @@ def index():
 
 @app.route('/live')
 def live():
-    return render_template('live.html')
+    data = {}
+    subdirs = [d for d in os.listdir(predict_dir)
+               if os.path.isdir(os.path.join(predict_dir, d))]
+
+    if not subdirs:
+        return render_template('live.html', nopredictions=True)
+
+    subdirs.sort()
+    latest_subdir = subdirs[-1]
+    latest_prediction = None
+    latest_img = None
+
+    for subdir in subdirs:
+        data[subdir] = []
+        jsons = os.listdir(os.path.join(predict_dir, subdir))
+        jsons.sort()
+
+        if subdir == latest_subdir:
+            latest_prediction = os.path.join(predict_dir, subdir, jsons[-1])
+            name = jsons[-1].replace('.json', app.config['IMAGE_SUFFIX'])
```

Comment:
This may be possibly unsafe, `somefile.json.json` gets replaced to `somefile.png.png`, provided that `app.config['IMAGE_SUFFIX']` is set to `.png`.

I believe doing a `re.sub` with anchoring to the end of string (`$`) would be much safer, though not strictly necessary. Do you feel like it does not make much sense to do that here?